src/App.tsx

```diff
@@ -1,61 +1,137 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const getTodos: Todo[] = todosFromServer.map(todo => ({
```

Comment:
It isn't function. 
Use more descriptive names.
Do you need to export this array?

```suggestion
const preparedTodos: Todo[] = todosFromServer.map(todo => ({
```